africanlii/admin.py

```diff
@@ -0,0 +1,27 @@
+from countries_plus.models import Country
+from django.contrib import admin
+
+from africanlii.models import Ratification, RatificationCountry
+from peachjam.models import Work
+
+
+class RatificationCountryAdmin(admin.TabularInline):
+    model = RatificationCountry
+    extra = 1
+
+    def formfield_for_foreignkey(self, db_field, request, **kwargs):
+        if db_field.name == "country":
+            kwargs["queryset"] = Country.objects.filter(continent="AF")
+        return super().formfield_for_foreignkey(db_field, request, **kwargs)
+
+
+@admin.register(Ratification)
+class RatificationAdmin(admin.ModelAdmin):
+    inlines = (RatificationCountryAdmin,)
+
+    def formfield_for_foreignkey(self, db_field, request, **kwargs):
+        if db_field.name == "work":
+            kwargs["queryset"] = Work.objects.filter(
+                documents__doc_type="legal_instrument"
```

Comment:
We mentioned this yesterday -- it's better to not limit the choices because there are inevitable edge cases. Any work should be ratifiable.

Reply:
right. removed